### DISCUSSION TITLE
The ability to train with small datasets with high accuracy

### DISCUSSION THREAD

**Discussion by poyaii** [deepmodeling/deepmd-kit#3598] [Q&A]
Hello. Can I train a potential with a radius of 6 angstrom for a small structure with three different atoms, a lattice size of 8 * 7 * 7 angstrom, and 54 atoms?! Does the DeepMD package have the capability to work with small datasets and achieve acceptable accuracy? I would appreciate it if you could guide me or provide an example of a small dataset trained with this package. Thank you.

**Comment by wanghan-iapcm** [comment]
Training with small cells is valid in deepmd-kit.

**Reply by poyaii** [reply]
Thank you, wanghan. Could you give me an example of a small structure that has been trained? Does the cutoff radius have to be larger than half of the big box?  Because I considered the cutoff radius larger than half of the box and my structure collapsed! ? Thank you.

**Reply by wanghan-iapcm** [reply]
no restriction on the size of cell.
